### ISSUE TITLE
Evaluate windows arm with WPF and cswinrt

### ISSUE THREAD

**Issue by wli3** [dotnet/sdk#12413]
We need to block/enable/revert Windows desktop and cswinrt since WPF is not present on win-arm

**Comment by driver1998** [comment]
It is fine to have WinForms only in the first Preview, no need to block it.
On the other hand, we want to get SDK going ASAP.

**Comment by sfoslund** [comment]
@wli3 this is the current error for the tCanPublishArm64Winforms test
```
 C:\code\sdk\artifacts\bin\redist\Debug\dotnet\sdk\5.0.100-dev\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(387,5): error NETSDK1082: There was no runtime pack for Microsoft.WindowsDesktop.App.WindowsForms available for the specified RuntimeIdentifier 'win-arm64'. [C:\code\sdk\artifacts\tmp\Debug\ItCanPublishA---4AB2DB53\ItCanPublishA---4AB2DB53.csproj]
```

**Comment by wli3** [comment]
@sfoslund thanks! I think this behavior is ok 5.0. Since it mentions "win-arm". We should fix it in 5.0.2xx

**Comment by sfoslund** [comment]
Sounds good, I've made a PR to remove this test since it's not relevant to 5.0 for now: https://github.com/dotnet/sdk/pull/13791

**Comment by JensNordenbro** [comment]
@wli3 ; Is this fixed in "5.0.2xx" ? 
I tried now but it does not publish to win10-arm64, however it did last friday(could have been me)

**Comment by marcpopMSFT** [comment]
I believe that 5.0.3xx and onward should have arm64 support for wpf and winforms.
